src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employeesContainer = document.querySelector('ul');
+const employees = [...document.querySelectorAll('[data-salary]')];
+const employeesList = employees.map((employee) => {
+  return {
+    name: employee.innerText,
+    position: employee.dataset.position,
+    salary: +employee.dataset.salary.replace(/[$,]/g, ''),
```

Comment:
The replacement of salary format using regex is concise but consider handling cases where the data attribute might not be set or is incorrectly formatted. This could lead to unexpected NaN values.

Reply:
bot are right, change your regex